drivers/couchbase/couchbase-driver/src/main/java/io/mongock/driver/couchbase/repository/CouchbaseRepositoryBase.java

```diff
@@ -51,7 +51,7 @@ public CouchbaseRepositoryBase(Cluster cluster, Collection collection, Set<Strin
   @Override
   public JsonObject mapFieldInstances(List<FieldInstance> fieldInstanceList) {
     JsonObject document = JsonObject.create();
-    document.put(DOCUMENT_TYPE_KEY, DOCUMENT_TYPE);
+    document.put(DOCUMENT_TYPE_KEY, DOCUMENT_TYPE_CHANGE_ENTRY);
```

Comment:
This ChangeEntry specific code should be moved to `CouchbaseChangeEntryRepository`.
Additionally, for LockEntry entity we should add another document type, named "mongockLockEntry", and assign it in its repository `CouchbaseLockRepository`.